test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Singleton/SingletonTest.cs

```diff
@@ -374,7 +427,7 @@ public async Task SingletonDerivedTypeTest(string model, string singletonName, s
         [InlineData("conAttr", "MonstersInc")]
         public async Task SingletonQueryOptionsTest(string model, string singletonName)
         {
-            string requestUri = string.Format(this.BaseAddress + "/{0}/{1}", model, singletonName);
+            string requestUri = string.Format(this.BaseAddress + "/{0}/{1}", model, singletonName); // "http://LAPTOP-4K4FHFF3:11003/conAttr/MonstersInc"
```

Comment:
different people have different machine name.
So, the comment here doesn't make sense. Remove it.

Reply:
✅